.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
 script: bash -ex .travis-docker.sh
 services:
   - docker
 sudo: false
 env:
  global:
-   - PACKAGE="publish"
+   - PACKAGE="opam-publish"
+   - EXTRA_ENV="OPAMSOLVERTIMEOUT=600"
    - PRE_INSTALL_HOOK="cd /home/opam/opam-repository && git pull origin master && opam update -u -y"
  matrix:
-   - DISTRO=debian-stable OCAML_VERSION=4.02.3
-   - DISTRO=debian-testing OCAML_VERSION=4.03.0
-   - DISTRO=debian-unstable OCAML_VERSION=4.04.0
-   - DISTRO=ubuntu-12.04 OCAML_VERSION=4.02.3
-   - DISTRO=ubuntu-16.04 OCAML_VERSION=4.04.0
-   - DISTRO=centos-6 OCAML_VERSION=4.04.0
-   - DISTRO=centos-7 OCAML_VERSION=4.03.0
-   - DISTRO=fedora-24 OCAML_VERSION=4.02.3
-   - DISTRO=alpine OCAML_VERSION=4.04.0
+   - DISTRO=debian-testing OCAML_VERSION=4.05
+   - DISTRO=debian-unstable OCAML_VERSION=4.05
+   - DISTRO=ubuntu-16.04 OCAML_VERSION=4.04
+   - DISTRO=ubuntu-18.04 OCAML_VERSION=4.05
+   - DISTRO=centos-8 OCAML_VERSION=4.05
+   - DISTRO=fedora-27 OCAML_VERSION=4.05
+   - DISTRO=fedora-28 OCAML_VERSION=4.06
+   - DISTRO=fedora-29 OCAML_VERSION=4.07
+   - DISTRO=fedora-30 OCAML_VERSION=4.07
+   - DISTRO=fedora-31 OCAML_VERSION=4.08
```

Comment:
```suggestion
   - DISTRO=fedora-30 OCAML_VERSION=4.08
```

Reply:
Regarding this change, I sent a PR adding both Fedora 31 and Ubuntu 19.10 https://github.com/avsm/ocaml-dockerfile/pull/16

Reply:
Same comment https://github.com/ocaml/opam-publish/pull/85#discussion_r351460044